api/src/gameState/gameState.service.ts

```diff
@@ -71,10 +73,21 @@ export class GameStateService {
     });
   }
 
-  async deleteGame(where: Prisma.GameWhereUniqueInput): Promise<Game> {
-    return this.prisma.game.delete({
+  async deleteGame(where: Prisma.GameWhereUniqueInput): Promise<{data: any}> {
+    const deletedGame = await this.prisma.game.delete({
       where,
     });
+    return { data: deletedGame };
+  }
+
+  async deleteManyGame(ids: number[]) {
+    const deletedGames = await this.prisma.game.deleteMany({
+      where: {
+        id: { in: ids },
+      },
+    });
+
+    return { data: [deletedGames] };
```

Comment:
Not sure why you don't return `deletedGames`, also, if you really want to return like this, you are returning it wrong, you return an array of array.